.github/workflows/dockerhub_sshnpd.yml

```diff
@@ -133,8 +154,27 @@ jobs:
           docker buildx imagetools create \
             -t atsigncompany/${{ matrix.name }}:${{ env.TAG2 }} \
             --append atsigncompany/${{ matrix.name }}:amd64-${{ env.TAG2 }}
+      - name: Save digest to file and sign combined manifests
```

Comment:
As we're doing a matrix build we can't just pass digests between jobs. We need to save them to a file, aggregate them, then use the aggregate for a matrix definition.